slurm_ops_manager/slurm_ops_base.py

```diff
@@ -331,6 +341,23 @@ def write_munge_key(self, munge_key):
         key = b64decode(munge_key.encode())
         self._munge_key_path.write_bytes(key)
 
+    def write_jwt_rsa(self, jwt_rsa):
+        """Write the jwt_rsa key."""
+
+        # Remove jwt_rsa if exists.
+        if self._jwt_rsa_key_file.exits():
```

Comment:
This might be overkill for our use case, but i'd write some random stuff in the file before deleting it:
```python
from Crypto.Random.random import getrandbits
foo.write_text(getrandbits(2048))
```